impl/pom.xml

```diff
@@ -27,17 +27,23 @@
    <modelVersion>4.0.0</modelVersion>
 
    <parent>
-        <groupId>org.glassfish</groupId>
+        <groupId>com.sun.faces</groupId>
         <artifactId>mojarra-parent</artifactId>
-        <version>4.0.1-SNAPSHOT</version>
-    </parent>
+        <version>4.0.0.SP01-SNAPSHOT</version>
+   </parent>
 
-   <artifactId>jakarta.faces</artifactId>
```

Comment:
I think `<groupId>com.sun.faces</groupId>` should be added here too.

Reply:
I can, but tf the groupId is the same as the parent, specifying it again is not necessary.

Reply:
Fixed